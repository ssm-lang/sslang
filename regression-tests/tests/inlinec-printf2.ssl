$$$
#include <stdio.h>

void printInt(int i, const char *s)
{
  printf("Hello, %d, %s!\n", i, s);
}
$$$

main _ _ =
  $printInt($$42$$, $$"world"$$)
