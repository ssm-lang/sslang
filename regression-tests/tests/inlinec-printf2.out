Hello, 42, world!
